x-pack/plugins/security_solution/public/hosts/pages/details/index.tsx

```diff
@@ -139,6 +134,17 @@ const HostDetailsComponent: React.FC<HostDetailsProps> = ({ detailName, hostDeta
 
   const isPlatinumOrTrialLicense = useMlCapabilities().isPlatinumOrTrialLicense;
 
+  const { hasKibanaREAD, hasIndexRead } = useAlertsPrivileges();
+  const canReadAlerts = hasKibanaREAD && hasIndexRead;
+
+  const entityFilter = useMemo(
+    () => ({
+      field: ES_HOST_FIELD,
+      value: detailName,
+    }),
+    [detailName]
+  );
```

Comment:
This `useMemo` is not very useful, we can pass the props directly to the `AlertCountByRuleByStatus`.

Reply:
However, it is weird that these values are passed in a different way to each component. We could pass `field` and `value` props (or `entityFilter` prop) to both components so they are uniform.

Reply:
fixed! went with entityFilter prop